spring-batch-core/src/main/resources/org/springframework/batch/core/migration/migration-hana.sql

```diff
@@ -0,0 +1,28 @@
+
+-- create the requisite table
+
+CREATE TABLE BATCH_JOB_EXECUTION_PARAMS  (
+	JOB_EXECUTION_ID BIGINT NOT NULL ,
+	TYPE_CD VARCHAR(6) NOT NULL ,
+	KEY_NAME VARCHAR(100) NOT NULL ,
+	STRING_VAL VARCHAR(250) ,
+	DATE_VAL TIMESTAMP DEFAULT NULL ,
+	LONG_VAL BIGINT ,
+	DOUBLE_VAL DOUBLE ,
+	IDENTIFYING VARCHAR(1) NOT NULL ,
+	constraint JOB_EXEC_PARAMS_FK foreign key (JOB_EXECUTION_ID)
+	references BATCH_JOB_EXECUTION(JOB_EXECUTION_ID)
+);
+
+-- insert script that 'copies' existing batch_job_params to batch_job_execution_params
+-- sets new params to identifying ones
+-- verified on h2, 
```

Comment:
> verified on h2

Is there a way to verify DDL scripts on a real SAP HANA instance? We have introduced Docker-based integration tests to verify DDL scripts on non-embeddable databases (see [MySQLJobRepositoryIntegrationTests](https://github.com/spring-projects/spring-batch/blob/main/spring-batch-core/src/test/java/org/springframework/batch/core/test/repository/MySQLJobRepositoryIntegrationTests.java) as an example). I see there is a Docker image for SAP HANA express edition (which requires an DockerHub account, but that's not an issue). Could you please add a Docker-based test for HANA?

Reply:
I have added an integration test based on the SQL Server test. Since the [PR ](https://github.com/testcontainers/testcontainers-java/pull/3017)for adding HANA to testcontainers is still open I've added a simple version of the HANA container class to the test project.